src/login/translation/languages/sv.json

```diff
@@ -299,6 +299,8 @@
   "register.sub-heading": "Registrera din e-postadress för att skapa ditt eduID.",
   "register.toLogin": "Om du redan har ett eduID kan du logga in",
   "required": "Obligatorisk",
+  "blank space not allowed": "Mellanslag ej tillåtet",
+  "only allow letters": "Ett namn består endast av bokstäver",
```

Comment:
Should this match the Eng version and say: "Ett namn kan bara bestå av bokstäver"?